### PR TITLE
Mpanuszewska/govt params

### PR DESCRIPTION
**Pull request by mpanuszewska**

### CODE REVIEW

**Review by Sebenicz — APPROVED** [review]

File: app/src/main/java/pl/llp/aircasting/ui/view/fragments/search_follow_fixed_session/SearchLocationFragment.kt [hunk 1]
```diff
@@ -73,11 +73,72 @@ class SearchLocationFragment @Inject constructor(
...
+            else -> StringConstants.airbeam2sensorName
+        }
+    }
+
+    private fun onChipGroupSecondLevelTwoSelected(chipGroup: ChipGroup) {
+        if (chipGroup.checkedChipId == binding.govtSecondChip.id) {
+            if (txtSelectedParameter == StringConstants.measurementTypeOzone) {
+                txtSelectedSensor = StringConstants.governmentSensorNameOzone
+            } else {
+                txtSelectedSensor = StringConstants.governmentSensorNameNitrogenDioxide
```
Comment (xameYeoh) [inline app/src/main/java/pl/llp/aircasting/ui/view/fragments/search_follow_fixed_session/SearchLocationFragment.kt:137]: Small thing, but in kotlin you can use if as an assignment operation:
`txtSelectedSensor = if (txtSelectedParameter == StringConstants.measurementTypeOzone) {
                StringConstants.governmentSensorNameOzone
            } else {
                StringConstants.governmentSensorNameNitrogenDioxide
            }`

Comment (mpanuszewska) [reply app/src/main/java/pl/llp/aircasting/ui/view/fragments/search_follow_fixed_session/SearchLocationFragment.kt:137]: The IDE even suggested to do that so it was a decision on my part. Though, if it's an actually used standars in kotlin, I'd be happy to review my opinion on it :D

Comment (xameYeoh) [reply app/src/main/java/pl/llp/aircasting/ui/view/fragments/search_follow_fixed_session/SearchLocationFragment.kt:137]: I also like to use "when" if I have two "if"s, even though there is only one case here, to not have nested "if"s:
`when (chipGroup.checkedChipId) {
            binding.govtSecondChip.id -> {
                txtSelectedSensor =
                    if (txtSelectedParameter == StringConstants.measurementTypeOzone) {
                        StringConstants.governmentSensorNameOzone
                    } else {
                        StringConstants.governmentSensorNameNitrogenDioxide
                    }
            }
        }`
        
But this is just stylistic thing

**Review by xameYeoh — APPROVED** [review]

File: app/src/main/java/pl/llp/aircasting/ui/view/fragments/search_follow_fixed_session/SearchLocationFragment.kt (hunk 1 above)
Comment (xameYeoh) [reply app/src/main/java/pl/llp/aircasting/ui/view/fragments/search_follow_fixed_session/SearchLocationFragment.kt:137]: > The IDE even suggested to do that so it was a decision on my part. Though, if it's an actually used standars in kotlin, I'd be happy to review my opinion on it :D

Yeah, I think most code suggestions by the IDE are based on the standards. Not all standards are there yet, but they try to keep track of most of them and ease the development

Comment (xameYeoh) [reply app/src/main/java/pl/llp/aircasting/ui/view/fragments/search_follow_fixed_session/SearchLocationFragment.kt:137]: > I also like to use "when" if I have two "if"s, even though there is only one case here, to not have nested "if"s:
> 
> ```kotlin
> when (chipGroup.checkedChipId) {
>             binding.govtSecondChip.id -> {
>                 txtSelectedSensor =
>                     if (txtSelectedParameter == StringConstants.measurementTypeOzone) {
>                         StringConstants.governmentSensorNameOzone
>                     } else {
>                         StringConstants.governmentSensorNameNitrogenDioxide
>                     }
>             }
>         }
> ```
> 
> But this is just stylistic thing

Don't ask me about nested "when"s though... :D

**Pull request merged**